### ISSUE TITLE
Feature Request: Make 'required' fields configurable or optional

### ISSUE THREAD

**Issue by BallisticPain** [jessepollak/card#354]
I realize that this originally was intended to fill in all the information available on a credit card and display it in a very nice manner. There are many applications where the full data isn't needed, and instead of having to hide controls on the page, it would be more advantageous to either be able to specify what's being required, or just most auxiliary items are optional.

If we're interesting in going with the latter option, I would vote that only the cc-number be required, and the rest are optional. I could potentially be persuaded to consider cc-expiry as a required field as well. Outside of that, though I believe the remainder of the fields should be optional. We have another ticket requesting something similar... #351

Anyone else want to chime in?

**Comment by SublymeRick** [comment]
I agree with this, just make the cc number required, you gotta think there may be a case where someone just needs someone to confirm a credit card number or something to that effect. You dont want to force developers to use this is specific ways, let the developer decide how he wants to use the plugin.
